eventkitd/utils/delimited/delimited_test.go

```diff
@@ -0,0 +1,117 @@
+package delimited
```

Comment:
how do you feel about private?

Reply:
Only, if we need to expose it somewhere. But, yes, private is better if we don't intend to provide API compatibility.

Reply:
I like `internal` better because it actually makes the code private and doesn't allow importers to access to logic until we're ready to publish and finalize the API

Reply:
lol that's actually the precise reason i don't like internal lol. it forces someone who can very definitely use the code if they want to to fork it.